examples/eks-cluster-with-observability/main.tf

```diff
@@ -0,0 +1,174 @@
+  zone        = "observability" # Environment within one sub_tenant or business unit
+
+  kubernetes_version = "1.21"
+
+  vpc_cidr     = "10.0.0.0/16"
+  vpc_name     = join("-", [local.tenant, local.environment, local.zone, "vpc"])
+  cluster_name = join("-", [local.tenant, local.environment, local.zone, "eks"])
+
+  terraform_version = "Terraform v1.1.4"
+}
+
+module "aws_vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "v3.11.3"
+
+  name = local.vpc_name
+  cidr = local.vpc_cidr
+  azs  = data.aws_availability_zones.available.names
+
+  public_subnets  = [for k, v in slice(data.aws_availability_zones.available.names, 0, 3) : cidrsubnet(local.vpc_cidr, 8, k)]
+  private_subnets = [for k, v in slice(data.aws_availability_zones.available.names, 0, 3) : cidrsubnet(local.vpc_cidr, 8, k + 10)]
+
+  enable_nat_gateway   = true
+  create_igw           = true
+  enable_dns_hostnames = true
+  single_nat_gateway   = true
+
+  public_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/elb"                      = "1"
+  }
+
+  private_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/internal-elb"             = "1"
+  }
+
+}
+
+module "aws-eks-accelerator-for-terraform" {
+  source = "../.."
+
+  tenant            = local.tenant
+  environment       = local.environment
+  zone              = local.zone
+  terraform_version = local.terraform_version
+
+  # EKS Cluster VPC and Subnet mandatory config
+  vpc_id             = module.aws_vpc.vpc_id
+  private_subnet_ids = module.aws_vpc.private_subnets
+
+  # EKS CONTROL PLANE VARIABLES
+  create_eks         = true
+  kubernetes_version = local.kubernetes_version
+
+  # EKS MANAGED NODE GROUPS
+  managed_node_groups = {
+    mg_4 = {
+      node_group_name = "managed-ondemand"
+      instance_types  = ["m5.xlarge"]
+      min_size        = 3
+      subnet_ids      = module.aws_vpc.private_subnets
+    }
+  }
+
+  # Enable Amazon Prometheus - Creates a new Workspace id
+  enable_amazon_prometheus = true
+}
+
+resource "aws_iam_policy" "fluentbit-opensearch-access" {
+  name        = "fluentbit-opensearch-access"
+  description = "IAM policy to allow Fluentbit access to OpenSearch"
+  policy      = data.aws_iam_policy_document.fluentbit-opensearch-access.json
+}
+
+module "kubernetes-addons" {
+  source         = "../../modules/kubernetes-addons"
+  eks_cluster_id = module.aws-eks-accelerator-for-terraform.eks_cluster_id
+
+  #K8s Add-ons
+  enable_metrics_server     = true
+  enable_cluster_autoscaler = true
+  enable_argocd             = true
+
+  # Fluentbit
+  enable_aws_for_fluentbit        = true
+  aws_for_fluentbit_irsa_policies = [aws_iam_policy.fluentbit-opensearch-access.arn]
+  aws_for_fluentbit_helm_config = {
+    aws_for_fluentbit_cw_log_group_name = "/${module.aws-eks-accelerator-for-terraform.eks_cluster_id
```

Comment:
is this needed?

Reply:
The log group names argument isn't needed. I'll remove this.